docs/recording/validating.md

```diff
@@ -36,6 +36,15 @@ ros2 caret check_ctf -d <path-to-trace-data>
 - Solution
   - Set `LD_PRELOAD` before running the application (See [recording section](./recording.md))
 
+### `Trace data from a package built without caret-rclcpp was detected.`
+
+- Cause
+
+  - Some packages, which used in the target application, is not built with CARET/rclcpp
+
+- Solution
+  - If you are using pre-built binaries obtained from apt or other sources, rebuild the application including the target package
```

Comment:
I am hoping for a little more detail information about "including the target package".
What's the difference between an operation that causes a pattern that works as expected and one that does not?